I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 384, in script call
    python:
  File "game/script.rpy", line 384, in script call
    python:
  File "game/script.rpy", line 327, in script
    python:
  File "game/script.rpy", line 376, in <module>
    renpy.say(noa,reply.get("content"))
  File "renpy/common/00action_other.rpy", line 582, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/ui.rpy", line 14, in save_tts_audio
    with open(os.path.join(renpy.config.gamedir,f"/tts/{renpy.random.randint(1000, 9999)}.wav"),"wb") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'C:/tts/6034.wav'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 384, in script call
    python:
  File "game/script.rpy", line 384, in script call
    python:
  File "game/script.rpy", line 327, in script
    python:
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\ast.py", line 821, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 376, in <module>
    renpy.say(noa,reply.get("content"))
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\exports\sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\character.py", line 1455, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\character.py", line 1106, in do_display
    display_say(who,
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\character.py", line 803, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\core.py", line 3278, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\layout.py", line 1294, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\layout.py", line 1294, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\layout.py", line 1294, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\screen.py", line 793, in event
    rv = self.child.event(ev, x, y, st)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\layout.py", line 1294, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\layout.py", line 1523, in event
    rv = super(Window, self).event(ev, x, y, st)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\layout.py", line 288, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\layout.py", line 1294, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\behavior.py", line 1180, in event
    return handle_click(self.clicked)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\behavior.py", line 1101, in handle_click
    rv = run(action)
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\display\behavior.py", line 401, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 582, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/ui.rpy", line 14, in save_tts_audio
    with open(os.path.join(renpy.config.gamedir,f"/tts/{renpy.random.randint(1000, 9999)}.wav"),"wb") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'C:/tts/6034.wav'

Windows-10-10.0.19041 AMD64
Ren'Py 8.3.2.24090902
Ushio_Noa 0.1.0_beta
Thu Dec 26 22:57:59 2024
